pudl_scrapers/spiders/eia860m.py

```diff
@@ -0,0 +1,123 @@
+"""Spider for 860 M."""
+# -*- coding: utf-8 -*-
```

Comment:
These encoding strings are no longer necessary because we only work with Python 3.